corehq/apps/reports/management/commands/create_tableau_connection.py

```diff
@@ -0,0 +1,220 @@
+import json
+import logging
+import requests
+from django.core.management import BaseCommand
+
+from corehq.apps.reports.models import TableauConnection
+
+from settings import HQ_ACCOUNT_ROOT
+
+logger = logging.getLogger('create_tableau_connection')
+logger.setLevel('DEBUG')
+
+
+class RequestFailure(Exception):
+    pass
+
+
+TABLEAU_CONNECTED_APP_NAME = "commcarehq_embedding"
```

Comment:
Should this be configurable? If multiple domains create connections does this stay the same?